docs/source/quickstart/index.rst

```diff
@@ -0,0 +1,216 @@
+==================================
+Welcome to Avocado's Documentation
+==================================
+
+Avocado is a set of tools and libraries to help with automated testing.
+
+One can call it a test framework with benefits.  Native tests are written in
+Python and they follow the :mod:`unittest` pattern, but any executable can
+serve as a test.
+
+How it works?
```

Comment:
Even though this form is usually in colloquial sentences, I believe we should follow a more formal language, so I'd go with either: "How does it work?", or "How it works".